projectroles/utils.py

```diff
@@ -249,7 +249,7 @@ def get_project_app_links(
                             'projectroles:update',
                             kwargs={'project': project.sodar_uuid},
                         ),
-                        'label': 'Update Project',
+                        'label': f'Update {get_display_name(project.type)}',
```

Comment:
You need to supply `title=True` as in the original implementation to ensure the name is returned capitalized. This was present in the original implementation. Please test these features to ensure they appear correct before submitting a pr.

Reply:
I'm sorry, that I overlooked it! Now it should be fixed